rules/aws_cloudtrail_rules/aws_ec2_startup_script_change.py

```diff
@@ -0,0 +1,24 @@
+from panther_base_helpers import aws_rule_context, deep_get
+
+
+def rule(event):
+    if event.get("eventName") == "ModifyInstanceAttribute":
```

Comment:
Since this has two positive assertions, I would `and` this into a single conditional.